### ISSUE TITLE
CORS error while logging in on fresh installation (using vagrant)

### ISSUE THREAD

**Issue by swaroopkml96** [medtagger/MedTagger#802]
## Current Behavior

On fresh install using vagrant,

Backend is in a vagrant VM (10.0.0.99)
Frontend is in the host (localhost)

Logging in using the default admin account throws a CORS error in the browser(firefox) console. Specifically,
```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://10.0.0.99:51000/api/v1/auth/sign-in. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing)
```

## Expected Behavior

Login should succeed

## Steps to Reproduce the Problem

1. Install backend using Vagrant (vagrant up)
2. Run make run_workers, make run_rest and make run_websocket inside vagrant VM
3. Serve the frontend in the host (ng serve)

## Additional comment (optional)
Note that I'm behind a corporate firewall but have set the appropriate proxy in VagrantFile.

**Comment by jpowie01** [comment]
Sadly, I cannot reproduce your issue. It doesn't occur on my dev machine without any proxy.

It looks that requests try to pass through proxy even though they shouldn't (as you're working on a local setup). Did you set up `NO_PROXY` settings to follow **both** `localhost` and `10.0.0.99`?

**Comment by swaroopkml96** [comment]
Hi @jpowie01, the issue persisted even after turning off the proxy altogether. 
Other things I tried:

1. The cors-everywhere firefox extension
2. Self-signing a certificate and running both the frontend and the rest api over https
These didn't fix it either.

As you say, the requests to 10.0.0.99 were going over the proxy for some reason.
Unfortunately, we ended up giving up and switching out the network for an open one.
If you're facing the same issue, it's best to talk to your IT department.

Closing as this is not an issue with medtagger per-se.
